apps/server/src/controllers/integrationController.ts

```diff
@@ -222,6 +239,32 @@ export function dispatchFromAdapter(type: string, payload: unknown, source?: 'os
       return { topic: 'timer', payload: timer };
     }
 
+    // ontime/change/{eventID}/{propertyName}
+    case 'change': {
+      try {
+        if (params.length < 2) {
+          throw new Error('Invalid parameters');
+        }
+
+        if (!payload || typeof payload !== 'string') {
```

Comment:
this blocks every thing that is not a string, before getting to type coercion
allowed types in osc should at least be: int, float, string and boolean
and should properly test `payload === undefined` so that empty strings and FALSE is allowed

Reply:
hmmmm, that one is my bad, I thought OSC payloads were always strings for some reason, do we need to check for other types here? we already coerce the payload, and check for `null` there